### ISSUE TITLE
Sync a deleted read-only file causes permission denied error

### ISSUE THREAD

**Issue by MorrisJobke** [owncloud/client#4445]
* having userA and userB
* userA shares folder `abc` read-only to userB
* userA puts some files into `abc`
* userB syncs down folder `abc` (check if the filesystem permissions are set to read only)
* userA deletes a file from `abc`
* userB client tries to delete the file on the next sync run

### expected

* file gets properly deleted

### actual 

* file is still there and client says "Permission denied"

```
02-08 15:46:11:115 0x4d576e0 OCC::Folder::slotSyncFinished:   ** error Strings:  ("abc/some-file.txt: Zugriff verweigert")
```

cc @butonic 

00004682

client 2.1.0 build 5683 on Windows

**Comment by MorrisJobke** [comment]
As a side note: renaming a read-only folder works just fine.

**Comment by butonic** [comment]
some more log:

```

02-08 15:46:08:120 0x4d576e0 _csync_merge_algorithm_visitor: INSTRUCTION_NONE     client file: [...]
02-08 15:46:09:823 0x4d576e0 _csync_merge_algorithm_visitor: INSTRUCTION_REMOVE   client file: Shared/muggefug/boom/Green_eco_friendly_badge.eps
02-08 15:46:10:270 0x4d576e0 _csync_merge_algorithm_visitor: INSTRUCTION_NONE     client file: [...]
02-08 15:46:10:271 0x4d576e0 csync_reconcile: Reconciliation for local replica took 0.65 seconds visiting 2140 files.
02-08 15:46:10:271 0x4d576e0 _csync_merge_algorithm_visitor: INSTRUCTION_NONE     server file: [...]
02-08 15:46:10:951 0x4d576e0 csync_reconcile: Reconciliation for remote replica took 0.68 seconds visiting 2139 files.
02-08 15:46:10:952 0x4d576e0 csync_statedb_close: sqlite3_close=0
02-08 15:46:10:952 0x4d576e0 OCC::SyncEngine::slotDiscoveryJobFinished: <<#### Reconcile end ####################################################  11494
02-08 15:46:10:954 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\muggefug\boom\Green_eco_friendly_badge.eps"
02-08 15:46:10:955 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\muggefug\boom\Green_eco_friendly_badge.eps"
02-08 15:46:10:955 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\muggefug\boom\Green_eco_friendly_badge.eps"
02-08 15:46:10:961 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\owncloud\"
02-08 15:46:10:962 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\owncloud\"
02-08 15:46:10:963 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\owncloud\"
02-08 15:46:10:965 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\owncloud\support\"
02-08 15:46:10:965 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\owncloud\support\"
02-08 15:46:10:966 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\owncloud\support\"
02-08 15:46:10:967 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\muggefug\"
02-08 15:46:10:967 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\muggefug\"
02-08 15:46:10:967 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\muggefug\"
02-08 15:46:10:968 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\"
02-08 15:46:10:969 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\"
02-08 15:46:10:969 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\"
02-08 15:46:10:973 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\owncloud\support\github-issues\"
02-08 15:46:10:974 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\owncloud\support\github-issues\"
02-08 15:46:10:975 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\owncloud\support\github-issues\"
02-08 15:46:10:976 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\muggefug\boom\"
02-08 15:46:10:977 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\muggefug\boom\"
02-08 15:46:10:977 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:SYNC:C:\Users\jfd\s3.owncloud.com\Shared\muggefug\boom\"
02-08 15:46:10:978 0x4d576e0 OCC::SyncEngine::slotDiscoveryJobFinished: Permissions of the root folder:  "RDNVCK"
02-08 15:46:10:980 0x4d576e0 OCC::SyncJournalDb::commitInternal: void OCC::SyncJournalDb::commitInternal(const QString&, bool) Transaction commit  "post treewalk" and starting new transaction
02-08 15:46:10:980 0x4d576e0 OCC::SyncJournalDb::commitInternal: void OCC::SyncJournalDb::commitInternal(const QString&, bool) Transaction commit  "post stale entry removal" and starting new transaction
02-08 15:46:10:981 0x4d576e0 OCC::OwncloudPropagator::start: Using QNAM/HTTP parallel code path
02-08 15:46:10:981 0x4d576e0 OCC::SyncEngine::slotDiscoveryJobFinished: <<#### Post-Reconcile end ####################################################  11523
02-08 15:46:10:982 0x4d576e0 OCC::AbstractNetworkJob::start: !!! OCC::PropfindJob created for "https://s3.owncloud.com/owncloud" + "/"
02-08 15:46:11:039 0x4d576e0 OCC::Folder::slotSyncStarted: #### Propagation start #################################################### >>
02-08 15:46:11:040 0x4d576e0 OCC::SocketApi::slotUpdateFolderView: Not sending UPDATE_VIEW for "ownCloud" because status() is 3
02-08 15:46:11:044 0x4d576e0 OCC::ownCloudGui::slotSyncStateChange: Sync state changed for folder  "ownCloud" :  "Sync Running"
02-08 15:46:11:046 0x4d576e0 OCC::PropagateLocalRemove::start: "C:/Users/jfd/s3.owncloud.com/Shared/muggefug/boom/Green_eco_friendly_badge.eps"
02-08 15:46:11:047 0x4d576e0 OCC::OwncloudPropagator::localFileNameClash: CaseClashCheck for  "C:/Users/jfd/s3.owncloud.com/Shared/muggefug/boom/Green_eco_friendly_badge.eps"
02-08 15:46:11:047 0x4d576e0 OCC::SyncJournalErrorBlacklistRecord::update: This error is not blacklisted  0 false
02-08 15:46:11:047 0x4d576e0 OCC::SyncEngine::slotItemCompleted: void OCC::SyncEngine::slotItemCompleted(const OCC::SyncFileItem&, const OCC::PropagatorJob&) "Shared/muggefug/boom/Green_eco_friendly_badge.eps" INSTRUCTION_REMOVE 2 "Zugriff verweigert
"
02-08 15:46:11:048 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:ERROR:C:\Users\jfd\s3.owncloud.com\Shared\muggefug\boom\Green_eco_friendly_badge.eps"
02-08 15:46:11:048 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:ERROR:C:\Users\jfd\s3.owncloud.com\Shared\muggefug\boom\Green_eco_friendly_badge.eps"
02-08 15:46:11:049 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:ERROR:C:\Users\jfd\s3.owncloud.com\Shared\muggefug\boom\Green_eco_friendly_badge.eps"
02-08 15:46:11:054 0x4d576e0 OCC::SyncEngine::slotItemCompleted: void OCC::SyncEngine::slotItemCompleted(const OCC::SyncFileItem&, const OCC::PropagatorJob&) "Shared/muggefug/boom" INSTRUCTION_NONE 0 ""
02-08 15:46:11:054 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\Shared\muggefug\boom"
02-08 15:46:11:055 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\Shared\muggefug\boom"
02-08 15:46:11:055 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\Shared\muggefug\boom"
02-08 15:46:11:056 0x4d576e0 OCC::SyncEngine::slotItemCompleted: void OCC::SyncEngine::slotItemCompleted(const OCC::SyncFileItem&, const OCC::PropagatorJob&) "Shared/muggefug" INSTRUCTION_NONE 0 ""
02-08 15:46:11:056 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\Shared\muggefug"
02-08 15:46:11:056 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\Shared\muggefug"
02-08 15:46:11:057 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\Shared\muggefug"
02-08 15:46:11:060 0x4d576e0 OCC::SyncJournalFileRecord::SyncJournalFileRecord: OCC::SyncJournalFileRecord::SyncJournalFileRecord(const OCC::SyncFileItem&, const QString&) "C:/Users/jfd/s3.owncloud.com/Shared/owncloud/support/github-issues" Retrieved inode  58919 (previous item inode:  0 )
02-08 15:46:11:062 0x4d576e0 OCC::SyncJournalDb::setFileRecord: Filtered writing the etag of "Shared/owncloud/support/github-issues/" because it is a prefix of "Shared/owncloud/support/github-issues/"
02-08 15:46:11:063 0x4d576e0 OCC::SyncJournalDb::setFileRecord: "INSERT OR REPLACE INTO metadata (phash, pathlen, path, inode, uid, gid, mode, modtime, type, md5, fileid, remotePerm, filesize, ignoredChildrenRemote, contentChecksum, contentChecksumTypeId) VALUES (?1 , ?2, ?3 , ?4 , ?5 , ?6 , ?7,  ?8 , ?9 , ?10, ?11, ?12, ?13, ?14, ?15, ?16);" -2648804347006179464 37 "Shared/owncloud/support/github-issues" 58919 "1454927582" "2" "_invalid_" "00795962515de1956eeee" "SRDNVCK" 0 0 "" "" 0
02-08 15:46:11:064 0x4d576e0 OCC::SyncEngine::slotItemCompleted: void OCC::SyncEngine::slotItemCompleted(const OCC::SyncFileItem&, const OCC::PropagatorJob&) "Shared/owncloud/support/github-issues" INSTRUCTION_NONE 0 ""
02-08 15:46:11:064 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\Shared\owncloud\support\github-issues"
02-08 15:46:11:065 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\Shared\owncloud\support\github-issues"
02-08 15:46:11:065 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\Shared\owncloud\support\github-issues"
02-08 15:46:11:066 0x4d576e0 OCC::SyncJournalFileRecord::SyncJournalFileRecord: OCC::SyncJournalFileRecord::SyncJournalFileRecord(const OCC::SyncFileItem&, const QString&) "C:/Users/jfd/s3.owncloud.com/Shared/owncloud/support" Retrieved inode  83296 (previous item inode:  0 )
02-08 15:46:11:066 0x4d576e0 OCC::SyncJournalDb::setFileRecord: Filtered writing the etag of "Shared/owncloud/support/" because it is a prefix of "Shared/owncloud/support/github-issues/"
02-08 15:46:11:067 0x4d576e0 OCC::SyncJournalDb::setFileRecord: "INSERT OR REPLACE INTO metadata (phash, pathlen, path, inode, uid, gid, mode, modtime, type, md5, fileid, remotePerm, filesize, ignoredChildrenRemote, contentChecksum, contentChecksumTypeId) VALUES (?1 , ?2, ?3 , ?4 , ?5 , ?6 , ?7,  ?8 , ?9 , ?10, ?11, ?12, ?13, ?14, ?15, ?16);" 3533203360202247013 23 "Shared/owncloud/support" 83296 "1454927582" "2" "_invalid_" "00794394515de1956eeee" "SRDNVCK" 0 1 "" "" 0
02-08 15:46:11:067 0x4d576e0 OCC::SyncEngine::slotItemCompleted: void OCC::SyncEngine::slotItemCompleted(const OCC::SyncFileItem&, const OCC::PropagatorJob&) "Shared/owncloud/support" INSTRUCTION_NONE 0 ""
02-08 15:46:11:068 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\Shared\owncloud\support"
02-08 15:46:11:068 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\Shared\owncloud\support"
02-08 15:46:11:068 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\Shared\owncloud\support"
02-08 15:46:11:073 0x4d576e0 OCC::SyncJournalFileRecord::SyncJournalFileRecord: OCC::SyncJournalFileRecord::SyncJournalFileRecord(const OCC::SyncFileItem&, const QString&) "C:/Users/jfd/s3.owncloud.com/Shared/owncloud" Retrieved inode  257644 (previous item inode:  0 )
02-08 15:46:11:074 0x4d576e0 OCC::SyncJournalDb::setFileRecord: Filtered writing the etag of "Shared/owncloud/" because it is a prefix of "Shared/owncloud/Conferences/"
02-08 15:46:11:074 0x4d576e0 OCC::SyncJournalDb::setFileRecord: "INSERT OR REPLACE INTO metadata (phash, pathlen, path, inode, uid, gid, mode, modtime, type, md5, fileid, remotePerm, filesize, ignoredChildrenRemote, contentChecksum, contentChecksumTypeId) VALUES (?1 , ?2, ?3 , ?4 , ?5 , ?6 , ?7,  ?8 , ?9 , ?10, ?11, ?12, ?13, ?14, ?15, ?16);" 25940700801959443 15 "Shared/owncloud" 257644 "1454940588" "2" "_invalid_" "00794381515de1956eeee" "SRDNVCK" 0 1 "" "" 0
02-08 15:46:11:075 0x4d576e0 OCC::SyncEngine::slotItemCompleted: void OCC::SyncEngine::slotItemCompleted(const OCC::SyncFileItem&, const OCC::PropagatorJob&) "Shared/owncloud" INSTRUCTION_NONE 0 ""
02-08 15:46:11:076 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\Shared\owncloud"
02-08 15:46:11:076 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\Shared\owncloud"
02-08 15:46:11:078 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\Shared\owncloud"
02-08 15:46:11:079 0x4d576e0 OCC::SyncEngine::slotItemCompleted: void OCC::SyncEngine::slotItemCompleted(const OCC::SyncFileItem&, const OCC::PropagatorJob&) "Shared" INSTRUCTION_NONE 0 ""
02-08 15:46:11:079 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\Shared"
02-08 15:46:11:080 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\Shared"
02-08 15:46:11:080 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\Shared"
02-08 15:46:11:084 0x4d576e0 OCC::SyncEngine::slotItemCompleted: void OCC::SyncEngine::slotItemCompleted(const OCC::SyncFileItem&, const OCC::PropagatorJob&) "" INSTRUCTION_NONE 0 ""
02-08 15:46:11:084 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\"
02-08 15:46:11:084 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\"
02-08 15:46:11:085 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK:C:\Users\jfd\s3.owncloud.com\"
02-08 15:46:11:089 0x4d576e0 OCC::SyncJournalDb::walCheckpoint: void OCC::SyncJournalDb::walCheckpoint() took 0 msec
02-08 15:46:11:089 0x4d576e0 OCC::SyncJournalDb::commitInternal: void OCC::SyncJournalDb::commitInternal(const QString&, bool) Transaction commit  "All Finished." 
02-08 15:46:11:090 0x4d576e0 OCC::SyncEngine::finalize: CSync run took  11631
02-08 15:46:11:090 0x4d576e0 OCC::BandwidthManager::~BandwidthManager: virtual OCC::BandwidthManager::~BandwidthManager()
02-08 15:46:11:094 0x4d576e0 OCC::Folder::slotSyncFinished:  - client version 2.1.0 (build 5683)  Qt 5.4.0  SSL  OpenSSL 1.0.2d 9 Jul 2015
02-08 15:46:11:095 0x4d576e0 OCC::Folder::slotSyncFinished: -> SyncEngine finished without problem.
02-08 15:46:11:101 0x4d576e0 OCC::Folder::bubbleUpSyncResult: Processing result list and logging took  4  Milliseconds.
02-08 15:46:11:112 0x4d576e0 OCC::Folder::bubbleUpSyncResult: OO folder slotSyncFinished: result:  3
02-08 15:46:11:115 0x4d576e0 OCC::Folder::slotSyncFinished:   ** error Strings:  ("Shared/muggefug/boom/Green_eco_friendly_badge.eps: Zugriff verweigert
")
02-08 15:46:11:116 0x4d576e0 OCC::Folder::slotSyncFinished:     * owncloud csync thread finished with error
02-08 15:46:11:116 0x4d576e0 OCC::Folder::slotSyncFinished: the last 2 syncs failed
02-08 15:46:11:117 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:ERROR:C:\Users\jfd\s3.owncloud.com\"
02-08 15:46:11:117 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:ERROR:C:\Users\jfd\s3.owncloud.com\"
02-08 15:46:11:117 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:ERROR:C:\Users\jfd\s3.owncloud.com\"
02-08 15:46:11:118 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "UPDATE_VIEW:C:\Users\jfd\s3.owncloud.com\"
02-08 15:46:11:118 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "UPDATE_VIEW:C:\Users\jfd\s3.owncloud.com\"
02-08 15:46:11:119 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "UPDATE_VIEW:C:\Users\jfd\s3.owncloud.com\"
02-08 15:46:11:121 0x4d576e0 OCC::ownCloudGui::slotSyncStateChange: Sync state changed for folder  "ownCloud" :  "Error"
02-08 15:46:11:122 0x4d576e0 OCC::FolderWatcher::pathIsIgnored: * Ignoring file "C:/Users/jfd/s3.owncloud.com/.owncloudsync.log"
02-08 15:46:11:224 0x4d576e0 OCC::SocketApi::command_RETRIEVE_FILE_STATUS: void OCC::SocketApi::command_RETRIEVE_FILE_STATUS(const QString&, QIODevice*) "C:\Users\jfd\s3.owncloud.com\Shared\muggefug\boom\Green_eco_friendly_badge.ai"
02-08 15:46:11:226 0x4d576e0 OCC::FileSystem::getModTime: Could not get modification time for "C:/Users/jfd/s3.owncloud.com/Shared/muggefug/boom/Green_eco_friendly_badge.ai" with csync, using QFileInfo
02-08 15:46:11:227 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK+SWM:C:\Users\jfd\s3.owncloud.com\Shared\muggefug\boom\Green_eco_friendly_badge.ai"
02-08 15:46:11:228 0x4d576e0 OCC::SocketApi::command_RETRIEVE_FILE_STATUS: void OCC::SocketApi::command_RETRIEVE_FILE_STATUS(const QString&, QIODevice*) "C:\Users\jfd\s3.owncloud.com\Shared\muggefug\boom\Green_eco_friendly_badge.eps"
02-08 15:46:11:230 0x4d576e0 OCC::FileSystem::getModTime: Could not get modification time for "C:/Users/jfd/s3.owncloud.com/Shared/muggefug/boom/Green_eco_friendly_badge.eps" with csync, using QFileInfo
02-08 15:46:11:230 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK+SWM:C:\Users\jfd\s3.owncloud.com\Shared\muggefug\boom\Green_eco_friendly_badge.eps"
02-08 15:46:11:233 0x4d576e0 OCC::SocketApi::command_RETRIEVE_FILE_STATUS: void OCC::SocketApi::command_RETRIEVE_FILE_STATUS(const QString&, QIODevice*) "C:\Users\jfd\s3.owncloud.com\Shared\muggefug\boom\Green_eco_friendly_badge.jpg"
02-08 15:46:11:235 0x4d576e0 OCC::FileSystem::getModTime: Could not get modification time for "C:/Users/jfd/s3.owncloud.com/Shared/muggefug/boom/Green_eco_friendly_badge.jpg" with csync, using QFileInfo
02-08 15:46:11:236 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK+SWM:C:\Users\jfd\s3.owncloud.com\Shared\muggefug\boom\Green_eco_friendly_badge.jpg"
02-08 15:46:11:237 0x4d576e0 OCC::SocketApi::command_RETRIEVE_FILE_STATUS: void OCC::SocketApi::command_RETRIEVE_FILE_STATUS(const QString&, QIODevice*) "C:\Users\jfd\s3.owncloud.com\Shared\muggefug\boom\Grundriss Neues Labor.svg"
02-08 15:46:11:238 0x4d576e0 OCC::FileSystem::getModTime: Could not get modification time for "C:/Users/jfd/s3.owncloud.com/Shared/muggefug/boom/Grundriss Neues Labor.svg" with csync, using QFileInfo
02-08 15:46:11:239 0x4d576e0 OCC::SocketApi::sendMessage: SocketApi:  Sending message:  "STATUS:OK+SWM:C:\Users\jfd\s3.owncloud.com\Shared\muggefug\boom\Grundriss Neues Labor.svg"
02-08 15:46:11:322 0x4d576e0 OCC::FolderMan::slotFolderSyncFinished: <===================================== sync finished for  "ownCloud"
```

ping me if you need more info, but initial reproduction steps should suffice. While renaming works, deleting does not. Same error for files and folders.

**Comment by ogoffart** [comment]
This is a known issue that is fixed in 2.1.1 already

**Comment by ogoffart** [comment]
#4277 and #4308

**Comment by MorrisJobke** [comment]
Thanks @ogoffart :) I just searched the client tracked and didn't found it :( Sorry for the noise.

**Comment by MorrisJobke** [comment]
And I know why - I only had the german translation and therefore searched for "Permission denied" and not "Access denied". :cry:

**Comment by cdamken** [comment]
Tested with 8.2.2 and 2.1.1 and the problem still appears

**Comment by cdamken** [comment]
Log and picture are in:

S3-ownCloud\Shared\owncloud\support\github-issues\client\4445

**Comment by cdamken** [comment]
We reinstall the server, if the problem still exist I will reopen the issue

**Comment by cdamken** [comment]
The problem appears again with 8.2.2 and 2.1.1

I asked for new logs

**Comment by cdamken** [comment]
The problem still exist and no information is shown in the logs

**Comment by ctsmike** [comment]
Problem still exists exactly as described in 2.1.1 build 5837.
I see the following in log of User B:
|0|/path/to/file.pdf|INST_NEW|Up|1398818804||358847||2|Not allowed because you don't have permission to add files in that folder|0|0|0|||INST_NONE|

**Comment by ogoffart** [comment]
`Not allowed because you don't have permission to add files in that folder`  Seems to be a geniune error message.

The bug was if there is only `Permission denied` Which comes from the file system.

I think you are seeing another bug.

**Comment by mafigo** [comment]
> Not allowed because you don't have permission to add files in that folder Seems to be a geniune error message.

>The bug was if there is only Permission denied Which comes from the file system.

>I think you are seeing another bug.

I Have the same problem and I think they are related, the nuance is that user B share on owncloud is read only. 
First you get the  "permission denied" error but since the file doesn't get deleted it still exists, the next time the client is restarted and scans it detects the undeleted as a missing file in the share and tries to synchronize it, but since the share is read only for user B  it gets the "Not allowed because you don't have permission to add files in that folder"  message.

Furthermore if user B deletes the file in the local machine you get an error because the client tries to delete the file in the share and it doesn't exist anymore, the only way I've been able to get pass the errors is to stop the client, manually delete the "offending" local files restart the client and do a full rescan pressing F5

**Comment by mafigo** [comment]
My previous post was inaccurate, because I didn't realize that the users(Win 8 and 10) experiencing this problem weren't updated to client 2.1.1, once updated, the permission denied issue was solved, I still had the  "Not allowed because you don't have permission to add files in that folder" message because the files were still there and to solve it I had to:

>  Furthermore if user B deletes the file in the local machine you get an error because the client tries to delete the file in the share and it doesn't exist anymore, the only way I've been able to get pass the errors is to stop the client, manually delete the "offending" local files restart the client and do a full rescan pressing F5

Sorry for the noise :(

**Comment by dragotin** [comment]
Thanks @mafigo that is good news.

**Comment by ctsmike** [comment]
Wait, was the bug fixed? Is there a build I can get to verify?
My problem still persists across all users on win 7, 8.1, 10

**Comment by ogoffart** [comment]
the problem is fixed in 2.1.1.

@ctsmike  your problem is different.   Please reopen a new bug report,  filling the issue template with the step to reproduce, expected behaviour and wrong behavior and log.

**Comment by ctsmike** [comment]
Will do... though the original description matches my problem to a T.
